config/checkstyle/checkstyle.xml

```diff
@@ -23,6 +23,7 @@
     </module>
 
     <module name="TreeWalker">
+        &prohibitJavaCollections;
```

Comment:
Now I assume that Java collections are prohibited by default, you need to opt-out from the prohibition (previously it was opt-in)